src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -590,221 +561,168 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
-    * @param mapContactIdContactOwnerChange a map of Contact Id to Contact, for the contacts to consider
-    * @param dmlWrapper to hold the Contacts that need updating.
+    * @param contactWithNewOwnerById a map of Contact Id to Contact, for the contacts to consider
     * @return void
-    */ 
-    private void updateOwners(map<Id, Contact> mapContactIdContactOwnerChange, DmlWrapper dmlWrapper) {        
-        if (mapContactIdContactOwnerChange.size() > 0) {
-            Map<Id, Account> allAccounts = new Map<Id, Account>();
-            Map<Id, Account> accountsToUpdate = new Map<Id, Account>();
+    */
+    private void updateOwners(Map<Id, Contact> contactWithNewOwnerById) {
+        if (contactWithNewOwnerById.isEmpty()) {
+            return;
+        }
 
-            for (Contact con : mapContactIdContactOwnerChange.values()) {
-                allAccounts.put(con.AccountId, null);
-            }
-                       
-            allAccounts = new Map<Id, Account>(
-                [SELECT Id, npe01__SYSTEM_AccountType__c, npe01__One2OneContact__c, OwnerId, 
-                    (SELECT Id FROM Contacts limit 2) 
-                 FROM Account WHERE Id IN :allAccounts.keySet()]
-            );
-            
-            for (Account acc : allAccounts.values()) {
-                // only consider 1:1 and HH Accounts
-                if (acc.npe01__SYSTEM_AccountType__c == CAO_Constants.ONE_TO_ONE_ORGANIZATION_TYPE || acc.npe01__SYSTEM_AccountType__c == CAO_Constants.HH_ACCOUNT_TYPE) {                    
-                    // if there is only 1 contact associated with this account
-                    if (acc.Contacts.size() == 1 && acc.npe01__One2OneContact__c != null) {
-                        Contact c = mapContactIdContactOwnerChange.get(acc.npe01__One2OneContact__c);
-                        Id ownerIdContact = (c != null ? c.OwnerId : null);
-                        if (acc.OwnerId != ownerIdContact) {
-                            acc.OwnerId = ownerIdContact;
-                            accountsToUpdate.put(acc.id, acc);
-                        }
-                    }
-                }   
-            }
+        Map<Id, Account> allAccounts = new Map<Id, Account>();
+        Map<Id, Account> accountsToUpdate = new Map<Id, Account>();
+
+        for (Contact con : contactWithNewOwnerById.values()) {
+            allAccounts.put(con.AccountId, null);
+        }
 
-            //update existing dmlWrapper objects with accounts already in the list
-            //avoids "duplicate id in list" errors updating the same account twice
-            for (sObject dmlObj : dmlWrapper.objectsToUpdate) {
-                Id objId = (Id) dmlObj.get('id');
-                if (accountsToUpdate.containsKey(objId)) {
-                    dmlObj.put('OwnerId', accountsToUpdate.get(objId).OwnerId);
-                    accountsToUpdate.remove(objId);
+        allAccounts = new Map<Id, Account>(
```

Comment:
I don't think `allAccounts` is used past the below `for` loop. Could the query just be in the `for` loop, and above where it populates the keys for `allAccounts` to use in the query maybe just use `set<Id>`.